src/setup-bot.ts

```diff
@@ -73,6 +78,29 @@ export async function setupBot (): Promise<void> {
     }),
   )
 
+  if (process.env['CHATOPERA_CLIENTID'] && process.env['CHATOPERA_SECRET']) {
+    const chatoperaFiltersGroup = process.env['CHATOPERA_FILTERS_GROUP']?.split(',')
+
+    haWechaty.use(
+      WechatyChatopera({
+        mention: false,
+        room: async (room: Room): Promise<boolean> => {
```

Comment:
Our Wechaty plugin [RoomMatcher](https://github.com/wechaty/wechaty-plugin-contrib/blob/16b69428cc486b49f5e0f897c0347ba30605df52/src/matchers/room-matcher.spec.ts#L122-L125) supports the following syntax: (with a RegExp list to match room topics)

```ts
room: [
  new RegExp(topicFilter1),
  new RegExp(topicFileter2),
],
```

The above code can be more readable and easy to be understand by replacing the blow code (a `matchFunction`)

Reply:
Since OSSChat supports multi tenants, the messaging routes are defined with profiles in https://github.com/kaiyuanshe/osschat/blob/7a7d58fa5a6d659e97a4436bd8d85e786ede9085/src/config-projects. 

So, there is no need of this group filters any more. Only groups defined in the profiles are handled auto reply with BOT.

Reply:
With Wechaty Chatopera Plugin, only the predefined groups in the `config-projects` are processed for auto reply.

Reply:
There might be some cases that the project admin wants to disable this FAQ feature.

If so, how can we deal with that?

Reply:
I think we should keep this discussion open because there are no solutions yet, thanks.

Reply:
PR https://github.com/kaiyuanshe/osschat/pull/155 does not deliver the FAQ features.
It contains -
* Collect chats into BOT, so BOT can analyze the history later.
* Enable dialogue to open issue automatically.

For the FAQ feature, it would be discuss and merge with following PR(s).

Reply:
Ok, got it. Thanks for the explaination!